curl_cffi/curl.py

```diff
@@ -69,11 +69,11 @@ def write_callback(ptr, size, nmemb, userdata):
     callback = ffi.from_handle(userdata)
     wrote = callback(ffi.buffer(ptr, nmemb)[:])
     wrote = ensure_int(wrote)
-    if wrote == CURL_WRITEFUNC_PAUSE or wrote == CURL_WRITEFUNC_ERROR:
+    if wrote in (CURL_WRITEFUNC_PAUSE, CURL_WRITEFUNC_ERROR):
```

Comment:
I know that `in (...)` is the preferred style, but this function is quite compute intencive, let's keep it the original form.